ci/qa/phpunit

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+cd $(dirname $0)/../../
+
+# PHPUnit Bridge should always be used in Symfony applications. (https://symfony.com/doc/current/components/phpunit_bridge.html)
+# This will create a phpunit executable in /bin/ instead of /vendor/bin/
+
+./vendor/bin/phpunit --configuration=ci/qa/phpunit.xml --coverage-text
```

Comment:
Is test coverage still generated? I did not see it in the output. Maybe because the deprecation warnings are overflooding it? It might be broken, because I removed the phpunit/phpcov dependency while upgrading to SF 4.4.

Reply:
Xdebug code coverage has now been enabled and is working properly.